docker-entrypoint.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
```

Comment:
I'm fine with removing all the pandoc/docker-related documentation files since I don't recall it being used anywhere. Could we also delete the [gh-pages branch](https://github.com/partiql/partiql-lang-kotlin/tree/gh-pages) (or create an issue to track deleting it) so that it's clearer we aren't supporting the pandoc/docker-related documentation.